src/stan-users-guide/simple_linear_regression/run.py

```diff
@@ -0,0 +1,35 @@
+""" Example program that generates data, compiles and runs regression.stan"""
```

Comment:
There's no regression.stan.  I'd suggest keeping titles more vague so they don't go stale.

Reply:
This is a problem, especially right now since the code is being gone over so many times. I'll try and keep things generic--good idea. This is why I went with run.R and run.py. I hesitate to make the stan programs generic but I'll defer. 

How about I do another footnote dir or two and see where we are at.